src/components/FormComponents/InputController/InputController.tsx

```diff
@@ -22,18 +20,28 @@ export const InputController = <T extends FieldValues>({
   maxLength,
   tooltip,
   InputProps,
+  defaultNumberValue = 1,
   ...textFieldProps
 }: InputControllerProps<T>) => {
   const { t } = useTranslation('app');
 
+  const getTextAdornment = (value: number) => {
+    if (!value) return null;
```

Comment:
if like that?

    if (textAdornment && value) {
      return <StyledBodyLarge>{t(textAdornment, { count: value })}</StyledBodyLarge>;
    }